patches/maintenance_theme.settings.patch

```diff
@@ -0,0 +1,2 @@
+// Setting to change the theme for maintenance page and update.php page.
```

Comment:
@rajeshreeputra Remove this file as it's not needed.

Reply:
removed maintenance_theme.settings.patch file